pkg/config/config.go

```diff
@@ -257,3 +267,25 @@ func asString(key string, target *string, values ...string) cm.ParseFunc {
 		return nil
 	}
 }
+
+// asStringSet parses the value at key as a sets.String (split by ',') into the target, if it exists.
+func asStringSet(key string, target *sets.String, values ...string) cm.ParseFunc {
+	return func(data map[string]string) error {
+		if raw, ok := data[key]; ok {
+			splitted := strings.Split(raw, ",")
+			if raw != "" {
```

Comment:
we can make this more readable by first checking

```
if raw == "" {
   // set empty target
   return nil
}
 ```

Reply:
changed.